docs/guide/packages/lucide-static.md

```diff
@@ -39,64 +53,77 @@ bun add lucide-static
 
 :::
 
-### CDN
+## SVG Files
 
-```html
-<!-- SVG file for a single icon -->
-<img src="https://unpkg.com/lucide-static@latest/icons/house.svg" />
+You can use standalone SVG files or SVG sprites in several ways.
 
-<!-- Icon font -->
-<style>
-  @font-face {
-    font-family: 'LucideIcons';
-    src: url(https://unpkg.com/lucide-static@latest/font/Lucide.ttf) format('truetype');
-  }
-</style>
-```
-
-## Usage
+Check out our [codesandbox example](https://codesandbox.io/s/using-the-svg-sprite-lz1kk).
 
-Check out the [codesandbox examples](https://codesandbox.io/s/using-the-svg-sprite-lz1kk).
+### SVG file as image
 
-### SVG Files
+#### In HTML:
 
-#### SVG file as image
-
-To use it in for example html:
+::: code-group
 
-```html
+```html [Webpack]
```

Comment:
Why the specific `webpack` keyword? Maybe use the word `Bundler`?

Reply:
Because `~lucide-react` is a Webpack specific syntax.